src/test/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/IntegerCopyNumberSegmentCollectionUnitTest.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.collections;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.tools.copynumber.formats.metadata.SampleLocatableMetadata;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.IntegerCopyNumberSegment;
+import org.broadinstitute.hellbender.tools.copynumber.gcnv.IntegerCopyNumberState;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Unit test for {@link IntegerCopyNumberSegmentCollection}.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public class IntegerCopyNumberSegmentCollectionUnitTest extends GATKBaseTest {
```

Comment:
Thanks for adding these tests!  I have been a little lazy about testing these collections classes.

Reply:
:)